frontend/public/locales/zh/logs.json

```diff
@@ -0,0 +1,16 @@
+{
+  "Log stream ended.": "日志流结束。",
+  "Loading log...": "正在载入日志...",
+  "Log stream paused.": "日志流已暂停。",
+  "Log streaming...": "日志流...",
+  "Raw": "原始",
+  "Download": "下载",
+  "Collapse": "折叠",
+  "Expand": "扩展",
+  "An error occurred while retrieving the requested logs.": "在获取请求的日志时发生错误。",
+  "Retry": "Retry",
```

Comment:
"Retry" could be translated into Chinese.

Reply:
This was updated.